### PR TITLE
Orthographic ellipse.

### PR DESCRIPTION
**Pull request by pelson**
Improve the disk produced by an orthographic projection.

### CODE REVIEW

**Comment by pelson** [comment]
Ironically, this fixes all 100 random projection problems of #579. :wink:

**Comment by pelson** [comment]
I'm looking into the test failures.

**Comment by ajdawson** [comment]
>Ironically, this fixes all 100 random projection problems of #579. ￼

Almost, but not quite: row 7 column 10 is missing the fill for Antarctica! However, it does appear to have a very friendly side-effect.

File: lib/cartopy/tests/test_polygon.py [hunk 1]
```diff
@@ -192,8 +192,8 @@ def test_tiny_point_between_boundary_points(self):
         area = projected.area
         # Before fixing, this geometry used to fill the whole disk. Approx
         # 1.2e14.
-        self.assertTrue(2.4e9 < area < 2.5e9,
-                        msg='Got area {}, expecting ~2.4e9'.format(area))
+        self.assertTrue(81330 < area < 81340,
+                        msg='Got area {}, expecting ~81336'.format(area))
```
Comment (QuLogic) [inline lib/cartopy/tests/test_polygon.py]: Does changing the boundary really introduce a change of 5 orders of magnitude here?

File: lib/cartopy/tests/mpl/test_images.py [hunk 2]
```diff
@@ -89,7 +89,7 @@ def test_web_tiles():
     ax.coastlines()
 
 
-@ImageTesting(['image_nest'], tolerance=17)
+@ImageTesting(['image_nest'])
```
Comment (QuLogic) [inline lib/cartopy/tests/mpl/test_images.py]: Travis appears to disagree with this change.

File: lib/cartopy/tests/test_polygon.py (hunk 1 above)
Comment (pelson) [inline lib/cartopy/tests/test_polygon.py]: It was surprising to me too. I've visually checked it, and it all looks good. I guess the important point is that the full disk is 10 orders of magnitude larger than area, where before it was 5 orders of magnitude. Given that this geometry was right on the edge, the act of reducing the edge is reasonably going to have a reasonable impact on the overall area.

**Comment by pelson** [comment]
Thanks @ajdawson & @QuLogic. I think this is the last issue that falls with v0.12.0. I'm going to go ahead and build and upload the docs once this is in.

**Pull request merged**